src/Parser.js

```diff
@@ -93,15 +93,22 @@ export function parse (value, root, parent, rule, rules, rulesets, pseudo, point
 						if (character === 123)
 							if (offset === 0)
 								parse(characters, root, reference, reference, props, rulesets, length, points, children)
-							else
-								switch (atrule === 99 && charat(characters, 3) === 110 ? 100 : atrule) {
-									// d l m s
-									case 100: case 108: case 109: case 115:
-										parse(value, reference, reference, rule && append(ruleset(value, reference, reference, 0, 0, rules, points, type, rules, props = [], length, children), children), rules, children, length, points, rule ? props : children)
-										break
+							else {
+								switch (atrule) {
+									// c(ontainer)
+									case 99:
+										if (charat(characters, 3) === 110) break
+									// l(ayer)
+									case 108:
+										if (charat(characters, 2) === 97) break
 									default:
```

Comment:
this is a default case that's not the last one 😅 I tried to code-golf this as much as possible and this is my best version so far

I was testing the size increase on `dist/umd/stylis.js` (since it's already minified) using `wc -c` and `gzip -c | wc -c`. Those are results:
| ref     | wc -c | gzip -c \| wc -c |
|---------|-------|------------------|
| master  | 10571 | 4396             |
| 9c57efb | 10607 | 4413             |
| 2a519d3 | 10619 | 4409             |
| ebd509a | 10606 | 4406             |

Reply:
The idea behind the fix here is that for non-layer and non-container (this case was already covered) we "reset" `offset`. We do the same in the default case and that's the code falls through from those when it doesn't `break` into that non-last default

Then based on the `offset` (if it survived or not) the correct parsing invocation is selected